src/models/PostModel.ts

```diff
@@ -18,7 +18,7 @@ export class PostModel {
   @Column("text", { array: true })
   categories: string[];
 
-  @Column("numeric", { scale: 2 } )
```

Comment:
was this manually done or did we run a linter? why wasn't the linter run before?

Reply:
linter gets rid of the space, but not sure why this wasn't done earlier